### ISSUE TITLE
Full code

### ISSUE THREAD

**Issue by bencooper222** [incongruity/nineteen-original#1]
Is there any plan for the full source to be released here? It's been irritating me that useninteen.com doesn't have SSL.

**Comment by incongruity** [comment]
Hi -

We haven’t had much time to give nineteen more time in a while - both Kim Erwin and I are up to our ears in other projects/our day jobs - but to be honest SSL matters very little in this case as *no* data goes to the server. Given that, can you see a compelling argument for using SSL?

If that’s a significant issue, I’ll try to get time in the next month or two to get a certificate and setup the server to use it. 

I’ll agree, the world is at the point were SSL is viewed as table-stakes and I understand that... but nineteen was designed to one-up even that because so many researchers can’t even risk sending their data to the cloud.

**Comment by bencooper222** [comment]
Yeah, it's mostly the table stakes thing that's why I think it should happen. It would also load faster because HTTP/2 allows objects to be concurrently loaded. I'm not entirely sure how usenineteen is hosted but it should be decently easy to slap Cloudflare in front of it and be done with it. 

Regardless, I agree - it's probably not the biggest priority in life 👍
